NitroxClient/Helpers/NitroxProtobufSerializer.cs

```diff
@@ -43,6 +46,14 @@ private void AddType(Type type)
             model.Add(type, true);
             knownTypes[type] = int.MaxValue; // UWE precompiled is going to pass everything to us
             NitroxTypes[type] = int.MaxValue;
+
+            if (type.IsSubclassOf(typeof(MonoBehaviour))) // Add Nitrox MonoBehaviours to the Component whitelist
+            {
+                if (!componentWhitelist.Contains(type.FullName))
+                {
+                    componentWhitelist.Add(type.FullName);
+                }
```

Comment:
`Add()` returns false if it already exists and won't throw exceptions. So it's redundant to check if it already contains the `string`.
```suggestion
                componentWhitelist.Add(type.FullName);
```

Reply:
Huh, must've confused it with adding to a Dictionary, throwing an exception when adding duplicates